pkg/pgmodel/pgx.go

```diff
@@ -274,12 +274,12 @@ func newPgxInserter(conn pgxConn, cache MetricCache, cfg *Cfg) (*pgxInserter, er
 }
 
 type pgxInserter struct {
+	insertedDatapoints     *int64
```

Comment:
this shouldn't matter, it's allocated separately, so the only thing that matters is that it's own allocation is aligned